src/Content/Backend/Solution/Monaco.Template.Backend.Domain.Tests/Factories/FixtureExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AutoFixture;
+using Monaco.Template.Backend.Domain.Tests.Factories.Entities;
+
+namespace Monaco.Template.Backend.Domain.Tests.Factories;
+
+public static class FixtureExtensions
+{
+	public static IFixture RegisterEntityFactories(this IFixture fixture) =>
+		fixture.RegisterCompany()
+			   .RegisterAddress()
+#if (!filesSupport)
+			   .RegisterCountry();
+#else
+			   .RegisterCountry()
+			   .RegisterDocument()
+			   .RegisterImage()
+			   .RegisterProduct();
+#endif
+
+	public static void RegisterMockFactories(this IFixture fixture) =>
+		fixture.RegisterCompanyMock()
+			   .RegisterAddressMock()
+#if (!filesSupport)
+			   .RegisterCountryMock();
+#else
+			   .RegisterCountryMock()
+			   .RegisterDocumentMock()
+			   .RegisterImage()
```

Comment:
Should this be using `RegisterImageMock` ?

Reply:
Same as with the `RegisterDocumentMock()` that I commented in the previous comment... Guess we can get rid of this one too...

Reply:
Fixed.